api/editor/api-editor/src/main/java/ca/bc/gov/tno/core/excel/CBRAReport.java

```diff
@@ -99,17 +122,23 @@ public XSSFWorkbook generateReport(ZonedDateTime from, ZonedDateTime to) {
 
     talkRadio = Arrays.stream(content).filter(c -> c.getMediaType().getName().equals("Talk Radio")).toList();
     talkRadioSeries = talkRadio.stream()
-        .collect(Collectors.groupingBy(c -> Optional.ofNullable((c.getSeries()))));
-    talkRadioKeys = new ArrayList<Optional<Series>>(talkRadioSeries.keySet());
+        .collect(Collectors.groupingBy(c -> c.getSeries() != null ? c.getSeries().getName() : "NOT SET"));
+    talkRadioKeys = new ArrayList<String>(talkRadioSeries.keySet());
 
     newsRadio = Arrays.stream(content).filter(c -> c.getMediaType().getName().equals("News Radio")).toList();
-    newsRadioSources = newsRadio.stream().collect(Collectors.groupingBy(Content::getSource));
+    newsRadioSources = newsRadio.stream().collect(Collectors.groupingBy(c -> {
+      if (c.getDataSource() != null)
+        return c.getDataSource().getCode();
+      if (c.getSource() != null && c.getSource().length() != 0)
+        return c.getSource();
+      return "NOT SET";
```

Comment:
We should never have `null` values for these in the database with good data, but I have to handle the scenario.  Currently I'm labelling them as **NOT SET**